### DISCUSSION TITLE
How to enable a check for staticcheck?

### DISCUSSION THREAD

**Discussion by ghost** [golang/vscode-go#1493] [Help] [answered]
I want to enable [ST1020](https://staticcheck.io/docs/checks#ST1020), what should I do?

**Comment by stamblerre** [comment, marked as answer]
Are you using `staticcheck` through the `gopls` configuration? If so, you can add the following to your settings:

```json5
"gopls": {
    "analyses": {
        "ST1020": true,
    }
}
```

**Reply by ghost** [reply]
This works, thank you!
